Material.Blazor/Components/BladeSet/MBBladeComponent.cs

```diff
@@ -27,9 +27,19 @@ public abstract class MBBladeComponent<TParam> : ComponentBase where TParam : MB
         [Parameter] public TParam Parameters { get; set; }
 
 
+        /// <summary>
+        /// Indicates whether the blade is open.
+        /// </summary>
+        public bool IsOpen { get; private set; } = true;
+
+
         /// <summary>
         /// A utility function to close the blade, calling BladeSet.RemoveBladeAsync(), passing the blade reference.
         /// </summary>
-        protected async Task CloseBladeAsync() => await BladeSet.RemoveBladeAsync(BladeReference).ConfigureAwait(false);
+        public async Task CloseBladeAsync()
+        {
+            IsOpen = false;
+            await BladeSet.RemoveBladeAsync(BladeReference).ConfigureAwait(false);
```

Comment:
You could also do `return BladeSet.RemoveBladeAsync(BladeReference)` which leaves any `ConfigureAwait` config to the caller.

Reply:
Good idea. That change is being committed shortly. I'd like to have a general `ConfigureAwait()` call with you for my own education.